e2e/test/scenarios/onboarding/search/search.cy.spec.js

```diff
@@ -290,18 +293,18 @@ describe("scenarios > search", () => {
       it("should hydrate created_by filter", () => {
         cy.intercept("GET", "/api/user").as("getUsers");
 
-        cy.request("GET", "/api/user/current").then(
-          ({ body: { common_name, id } }) => {
-            cy.visit(`/search?q=orders&created_by=${id}`);
+        cy.visit(`/search?q=orders&created_by=${NORMAL_USER_ID}`);
```

Comment:
I simplified the `created_by` and `last_edited_by` tests to use one question instead of 5 - I figured that there was no reason to check for 5 when we can check for 1.